src/pages/bug-bounty.tsx

```diff
@@ -11,225 +10,310 @@ import Card from "../components/Card"
 import Leaderboard from "../components/Leaderboard"
 import BugBountyCards from "../components/BugBountyCards"
 import Link from "../components/Link"
-import Emoji from "../components/OldEmoji"
+import Emoji from "../components/Emoji"
 import CardList from "../components/CardList"
 import Breadcrumbs from "../components/Breadcrumbs"
 import ButtonLink from "../components/ButtonLink"
 import PageMetadata from "../components/PageMetadata"
 import ExpandableCard from "../components/ExpandableCard"
-import {
-  CardContainer,
-  Content,
-  Divider,
-  Page,
-  GrayContainer,
-  GradientContainer,
-  SloganGradient,
-} from "../components/SharedStyledComponents"
 import FeedbackCard from "../components/FeedbackCard"
 import { Context } from "../types"
 import { getImage } from "../utils/image"
+import { Box, Center, Heading, Text } from "@chakra-ui/react"
```

Comment:
import order

Lets put 3rd party imports at the top of the import list, along with the `"gatsby"` or `"react"` imports.